### PR TITLE
Logging frameworks in python

### PR DESCRIPTION
**Pull request by PratikshaPatil77**
Web Page for GUIDE: Logging frameworks in python.

https://python-frameworks.netlify.app/

Above is Netlify link for this webpage.

### CODE REVIEW

File: guides/guides.css [hunk 1]
```diff
@@ -132,6 +132,83 @@ html {
     font-weight: 500;
     letter-spacing: 0.02rem;
 }
+.python-img1{
+    height: 350px;
```
Comment (nbhavana) [inline guides/guides.css:136]: When you add css in this guides.css file for any doc, add a comment that this block contains css for that particular doc

**Review by nbhavana — COMMENTED** [review]
The table does not look neat, can you make the header look better. Also make the text centered for all cells.

File: guides/guides.css (hunk 1 above)
Comment (PratikshaPatil77) [reply guides/guides.css:136]: Yeah Sorry I forget it . I will add it.

**Comment by PratikshaPatil77** [comment]
I'll centre the content in each cell, but could you kindly clarify what you want changed about the header?

**Comment by PratikshaPatil77** [comment]
@nbhavana @sonamgupta21 Changes are ready for review.

**Review by nbhavana — APPROVED** [review]

**Pull request merged**